README.md

```diff
@@ -7,6 +7,111 @@ The SDK is compatible with:
 * .NET Framework 4.6.1
 * Mono
 
+# Installation
```

Comment:
Can we make these H2s instead of H1s? I think it's more visually clear when the only H1 is at the very top of the readme.